ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -571,7 +571,7 @@ void Replica::handle(Request* m)
 
         if (rqueue.append(m))
         {
-          if (!limbo)
+          if (!limbo && f() > 0)
```

Comment:
Since we have turned off forwarding from CCF when we are running with PBFT, we should check that this doesn’t have any effect if members communicate with a backup instead of the primary before the network is opened.